packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -565,10 +571,7 @@ export class BuildService {
       },
       include: ENTITIES_INCLUDE
     });
-    return (orderBy(
-      entities,
-      entity => entity.createdAt
-    ) as unknown) as DataServiceGenerator.Entity[];
+    return orderBy(entities, entity => entity.createdAt);
```

Comment:
Really minor, but I'd love a line break before the `return` statement :)

Reply:
Done